src/pages/food/components/FoodDetailsPage.tsx

```diff
@@ -17,47 +19,119 @@ const TagList = styled.ScrollView`
   margin-bottom: ${({ theme: { space } }) => space.xl};
 `;
 
+const NutritionChart = styled(ProgressCircle)`
+  height: 100px;
+`;
+
 export type Props = {
   onBack?: () => void;
   onAdd?: () => void;
   foodDetails: GenericFood;
+  nutrition: Array<Nutrition>;
 };
 
 const FoodDetailsPage: React.FC<Props> = ({
-  onAdd,
   foodDetails: {
     name, tags,
   },
-}) => (
-  <Box height="100%" width="100%">
-    <Visual
-      size="large"
-      source={{
+  nutrition,
+}) => {
+  const carbs = nutrition ? nutrition.find((item) => item.name === "Carbs") : null;
+  const protein = nutrition ? nutrition.find((item) => item.name === "Protein") : null;
+  const fat = nutrition ? nutrition.find((item) => item.name === "Fat") : null;
+  const totalNutrition = nutrition ? nutrition.reduce((acc, item) => item.value + acc, 0) : null;
+
+  const carbsPercent = carbs && totalNutrition
+    ? parseInt(((carbs.value / totalNutrition) * 100).toFixed(0), 10) : 0;
+  const proteinPercent = protein && totalNutrition
+    ? parseInt(((protein.value / totalNutrition) * 100).toFixed(0), 10) : 0;
+  const fatPercent = fat && totalNutrition
+    ? parseInt(((fat.value / totalNutrition) * 100).toFixed(0), 10) : 0;
+
+  const { bottom } = useSafeArea();
+
+  return (
+    <Box height="100%" width="100%">
+      <Visual
+        size="large"
+        source={{
         uri:
-          'https://www.chiceats.com/sites/default/files/styles/image_1024x768/public/recipe/photo/homemade-pasta-recipe-1080x810@2x.jpg',
+        'https://www.chiceats.com/sites/default/files/styles/image_1024x768/public/recipe/photo/homemade-pasta-recipe-1080x810@2x.jpg',
       }}
-    />
-    <DrawerCard
-      topOffset={ 300 }
-      topRightOverlay={ (
-        <IconButton variant="warning" onPress={ onAdd }>
-          <AddIcon variant="inverted" />
-        </IconButton>
-      ) }
-    >
-      <Box px="l">
-        <Subtitle mb="s">PRODUCE</Subtitle>
-        <Title mb="s">{titleHelper(name)}</Title>
-      </Box>
-      <TagList horizontal showsHorizontalScrollIndicator={ false }>
-        {tags && tags.map((tag: TagType) => (
-          <Box key={ tag.id } alignSelf="flex-start" mr="xs">
-            <Tag value={ tag.name } />
-          </Box>
-        ))}
-      </TagList>
-    </DrawerCard>
-  </Box>
-);
+      />
+      <DrawerCard topOffset={ 300 }>
+        <Box px="l" mt="s">
+          <Subtitle mb="s">PRODUCE</Subtitle>
+          <Title mb="s">{titleHelper(name)}</Title>
+        </Box>
+        <TagList horizontal showsHorizontalScrollIndicator={ false }>
+          {tags && tags.map((tag: TagType) => (
+            <Box key={ tag.id } alignSelf="flex-start" mr="xs">
+              <Tag value={ tag.name } />
+            </Box>
+      ))}
+        </TagList>
+        <Grid>
+          <Column>
+            <NutritionChart progress={ carbsPercent / 100 } progressColor={ colors.neoncarrot } />
```

Comment:
The chart needs to be animated

Reply:
We might want to look into a new charts library. The animations look bad, plus they only animate when the values change, not when the page initializes

Reply:
do some research then

Reply:
sounds good

Reply:
I had trouble finding a charts library with the specific chart we needed + animations, so I used a built in react native thing called "Animated." The charts are now animated